src/System.Runtime.Loader/tests/Configurations.props

```diff
@@ -3,6 +3,7 @@
   <PropertyGroup>
     <BuildConfigurations>
       netcoreapp;
+      uap;
```

Comment:
the pattern is to have just netstandard if you don't need special uap code; else netstandard;uap if you don't need special uapaot code; else netstandard;uap;uapaot. Don't add uapaot unless you must have it.